src/components/load-button.vue

```diff
@@ -41,18 +49,23 @@
               }))
             )
           );
-  
+
           this.$root.map.fitBounds(bounds, { padding: 50 });
 
           this.shouldZoomIn = false;
         }
       },
     },
     methods: {
-      ...mapActions({
-        showError: 'notifications/showError',
-      }),
+      ...mapActions('data', [ 'loadProject', 'reset' ]),
+      ...mapActions('mapbox', [ 'getFeature' ]),
+      ...mapActions('selections', [ 'addSelection' ]),
+      onClick() {
+        this.$refs.uploader.click();
```

Comment:
Why is this needed?

Reply:
Instead of clicking on an 'invisible' input field to trigger the upload UI, where the buttons hover/focus effects would not show. We now use a button, have hover/focus styles, and when clicking on it we trigger a click on the 'hidden' input field.